authentication/graphql.py

```diff
@@ -135,11 +135,20 @@ def resolve_history(data, args, _context, _info):
     @staticmethod
     def resolve_balance(data, _args, _context, _info):
         """
-        Finds the current balance of the user
+        Finds a stock given a case insensitive name
         """
         client = plaid.Client(client_id=PLAID_CLIENT_ID, secret=PLAID_SECRET,
                               public_key=PLAID_PUBLIC_KEY, environment=PLAID_ENV)
-        balance = client.Accounts.get(data.access_token)['accounts'][0]['balances']['available']
+        balances = client.Accounts.balance.get(data.access_token)['accounts']
+        print(balances)
+        extracted_balances = [
+            (b['balances']['available']
+             if b['balances']['available'] is not None
+             else b['balances']['current'])
```

Comment:
This won't work :( Credit cards have balances that are positive, but they should be subtracted from your total balance. So you have to check if type is credit card, and apply a negative in that case. Nice list comprehension tho